### PR TITLE
added a back button at the top left of an article

### PR DESCRIPTION
**Pull request by TrevorOctober**
@miltontony @hodgestar Please review (https://praekeltorg.atlassian.net/browse/BC-9)

### CODE REVIEW

File: unicorecmsmama/templates/content.pt [hunk 1]
```diff
@@ -7,6 +7,9 @@
             <div class="h1" tal:condition="primary_category">
                 <a href="/content/list/${primary_category.uuid}/">${primary_category.title}</a>
             </div>
+             <div class="pagi" tal:condition="primary_category">
+                <a href="/">&lt;<span i18n:translate="Back">Back</span> </a>
```
Comment (miltontony) [inline unicorecmsmama/templates/content.pt]: This should link to `/content/list/${primary_category.uuid}` effectively 1 level up from the article.

**Comment by TrevorOctober** [comment]
@miltontony Ready for re-review

**Comment by TrevorOctober** [comment]
screenshot from 2016-03-15 13 46 45

**Comment by miltontony** [comment]
:+1:

**Pull request merged**